defaults/main.yaml

```
@@ -1,9 +1,45 @@
 ---
 mariadb:
+  system:
+  user:
+    root:
+      name: root
+      group: >-
+        {%- if ansible_system == 'Linux' -%}
+          root
+        {%- else -%}
+          wheel
+        {%- endif -%}
```

I think indentation is wrong here.
```suggestion
    user:
      root:
        name: root
        group: >-
          {%- if ansible_system == 'Linux' -%}
            root
          {%- else -%}
            wheel
          {%- endif -%}
```

---

No it's not. Because it is not the MariaDB user it is the operating sysem's user.

---

fixed in 780e32d36509e20ea7b735976cbeb7915e1ae8ae